### PR TITLE
Import dcos/dcos-cli@46463a3.

### PR DESCRIPTION
**Pull request by jeschkies**
Summary:
This patch imports the source code of the dcos-cli python library from
tag 0.5.7 (dcos/dcos-cli@46463a3). The CLI team moved over to a Go based
client and dropped support for the Python client. We only require a few
tools. This is the first step for refactoring.

JIRA issues: MARATHON-8384

### CODE REVIEW

File: tests/system/Pipfile.lock [hunk 1]
```diff
@@ -168,44 +188,27 @@
...
-                "sha256:d46f4e5d455cb5563685c52ef212696f0a6cc1ea627603218eabbd8a095291d8",
-                "sha256:d68b0cc40a8432ed3fc84876c519de704d6001800ec22b136e75ae841910c45b",
-                "sha256:da9291502cbc87dc0284a20c56876e4d2e68deac61cc43df4aec934e44ca97b1",
-                "sha256:e0fdb8322206fa02aa38f71519ff75dce2eb481b7e1110e2936795cb376bb6ee",
-                "sha256:eb8bb79d0ab00c931c8333b745f06fec481a51c52d70acd4ee95d6093ba5c386",
-                "sha256:fe859e445abc9ba9e97950ddafb904e23234c4ecb76b0fae6c86e80592ce464a"
-            ],
-            "version": "==2.0.2"
-        },
-        "dcos": {
```
Comment (jeschkies) [inline tests/system/Pipfile.lock]: That's the important bit here. `dcos` is import through Shakedown.

**Comment by jeschkies** [comment]
Once this patch landed we can unify `tests/shakedown/shakedown/dcos/marathon.py` and `tests/shakedown/dcos/marathon.py`. Confusing, huh?

**Comment by jeschkies** [comment]
Only our old friends `test_marathon_root.test_app_update_rollback` is failing.

**Comment by zen-dog** [comment]
> Once this patch landed we can unify tests/shakedown/shakedown/dcos/marathon.py and tests/shakedown/dcos/marathon.py. Confusing, huh?

Lol, you don't say

**Comment by jeschkies** [comment]
`strict` *does* fail!

**Comment by jeschkies** [comment]
Strict passes!

File: ci/si_install_deps.sh [hunk 2]
```diff
@@ -27,7 +27,18 @@ if ! command -v timeout >/dev/null 2>&1; then
...
         brew install coreutils
         PATH="/usr/local/opt/coreutils/libexec/gnubin:$PATH"
+        export PATH
+    fi
+fi
+
+# Ensure the latest DC/OS CLI is available
+if ! command -v dcos >/dev/null 2>&1; then
+    if [ "$PLATFORM" == 'Darwin' ]; then
+	curl -o /usr/local/bin/dcos https://downloads.dcos.io/binaries/cli/darwin/x86-64/dcos-1.11/dcos
```
Comment (jeschkies) [inline ci/si_install_deps.sh]: We still use the CLI but the latest version and it does *not* come with Shakedown since it should become a lib.

**Pull request merged**